site/src/pages/news/20220714-newsletter-4.mdx

```diff
@@ -0,0 +1,169 @@
+---
+date: 2022-07-14
+---
+
+# Armeria Newsletter vol. 4
+
+<video class="hideOnReducedMotion" src="../../images/armeria.m4v" preload="none" autoplay muted loop style="width: 282px; height: 112px;">
+  <img src="../../images/armeria.gif" loading="lazy" width="282" height="112" />
+</video>
+
+## From the devs
+
+<div class="noBullet">
+
+- ✨ We've released 7 new versions since our last newsletter with a bunch of new exciting features!
+  - [1.17.1](https://armeria.dev/release-notes/1.17.1/)
+    - Fixing a bug where Retrofit client calls hang indefinitely
```

Comment:
How about removing micro versions in the list, so we focus on new features only? I'm afraid people might stop reading if the list gets too large.